addons/mail/tests/test_mail_template.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="mail_template_test" model="mail.template">
```

Comment:
Could be great to have some other fields (email_from, attachment_ids for example, maybe not all but just a few) as classic templates and check they are reset, just to be close or standard use case.